validated/dash_apps/finished_apps/daily_validation.py

```diff
@@ -7,18 +7,21 @@
 import plotly.graph_objects as go
 from dash import Input, Output, State, dcc, html
 from dash_ag_grid import AgGrid
-from django.db.models import Max, Min
 from django_plotly_dash import DjangoDash
 
-from measurement.models import Measurement
 from measurement.validation import (
     generate_validation_report,
     reset_validated_days,
     reset_validated_entries,
     save_validated_days,
     save_validated_entries,
 )
-from station.models import Station
+from validated.filters import (
+    populate_stations_dropdown_util,
+    populate_variable_dropdown_util,
+    set_date_range_util,
+    set_min_max_util,
```

Comment:
Out of curiosity, why the `_util` at the end?

If it is to avoid clashing names, it is more common to either `import something as something_`, which is convoluted when you import multiple things, or call the functions within the module the same way but starting with an underscore to indicate they are internal to the module. Something like:

```python
from filters import some_filter
def _some_filter(...):
    some_filter(...)
```